docs/source/interfaces/indirect/Indirect Corrections.rst

```diff
@@ -55,10 +55,10 @@ Options
 ~~~~~~~
 
 Sample
-  Either a reduced file (_red.nxs) or workspace (_red) or an S(Q,\omega) file (_sqw.nxs) or workspace (_sqw) that represents the sample.
+  Either a reduced file (_red.nxs) or workspace (_red), an S(Q,\omega) file (_sqw.nxs) or workspace (_sqw) or an ELF file (_elf.nxs) or workspace (_elf) that represents the sample.
 
 Container
-  Either a reduced file (_red.nxs) or workspace (_red) or an S(Q,\omega) file (_sqw.nxs) or workspace (_sqw) that represents the container.
+  Either a reduced file (_red.nxs) or workspace (_red), an S(Q,\omega) file (_sqw.nxs) or workspace (_sqw) or an ELF file (_elf.nxs) or workspace (_elf) that represents the container.
```

Comment:
Super nitpicky, I know, but could we have another comma here to separate out the list?
```suggestion
  Either a reduced file (_red.nxs) or workspace (_red), an S(Q,\omega) file (_sqw.nxs) or workspace (_sqw), or an ELF file (_elf.nxs) or workspace (_elf) that represents the container.
```